lib/rubocop/cop/rspec/repeated_block_description.rb

```diff
@@ -0,0 +1,74 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module RSpec
+      # Check for repeated describe and context block descriptions.
+      #
+      # @example
+      #
+      #    # bad
+      #    describe 'cool feature' do
+      #      # example group
+      #    end
+      #
+      #    describe 'cool feature' do
+      #      # example group
+      #    end
+      #
+      #    # bad
+      #    context 'when case x' do
+      #      # example group
+      #    end
+      #
+      #    describe 'when case x' do
+      #      # example group
+      #    end
+      #
+      #    # good
+      #    describe 'cool feature' do
+      #      # example group
+      #    end
+      #
+      #    describe 'another cool feature' do
+      #      # example group
+      #    end
+      #
+      #    # good
+      #    context 'when case x' do
+      #      # example group
+      #    end
+      #
+      #    describe 'when another case' do
+      #      # example group
+      #    end
+      #
+      class RepeatedBlockDescription < Cop
+        MSG = "Don't repeat block descriptions"
+
+        def_node_matcher :several_blocks?, <<-PATTERN
+          (begin (block (send nil? {:context :describe} ...) ...)+ )
```

Comment:
Please use `ExampleGroup::ALL` to cover `feature`/`example_group`/`fdescribe` etc.